pkg/minikube/cluster/cluster.go

```diff
@@ -129,6 +129,11 @@ func DeleteHost(api libmachine.API) error {
 	m := util.MultiError{}
 	m.Collect(host.Driver.Remove())
 	m.Collect(api.Remove(constants.MachineName))
+
+	if len(m.Errors) == 0 && glog.V(2) {
```

Comment:
Notice that by placing it here, the same applies to `minikube delete`. I just preferred this to make the VM deletion messages coherent, independently of the trigger.

Reply:
This is how it looks now and it does not look good as there is a repetition to about VM getting deleted. Also the whole output needs to be fixed as `Deleted: Minishift VM 'minishift'` should be `Minishift VM 'Minishift' deleted` to match it with other messages in `minishift delete`.  
```
$ minishift delete --v 5
You are deleting the Minishift VM: 'minishift'. Do you want to continue [y/N]?: y
Deleting the Minishift VM...
Deleted: Minishift VM 'minishift'
Minishift VM deleted.
```
However I would suggest we just change the `profile delete` command output in this PR.

Reply:
Right. I'm gonna move this back to `profile delete` only